src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;

```

Comment:
```suggestion
    private double radius;
```
Let’s  use double (check it in other classes)